src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

```diff
@@ -390,6 +390,10 @@
                 <source>This value should be a valid expression.</source>
                 <target>This value should be a valid expression.</target>
             </trans-unit>
+            <trans-unit id="101">
+                <source>This value is not a valid hexadecimal color.</source>
+                <target>This value is not a valid hexadecimal color.</target>
```

Comment:
In translation messages we should also replace `hexadecimal color` by `CSS color`